subsys/bluetooth/shell/bt.c

```diff
@@ -1777,6 +1777,34 @@ static int cmd_past_unsubscribe(const struct shell *shell, size_t argc,
 
 	return err;
 }
+
+static int cmd_per_adv_sync_transfer(const struct shell *shell, size_t argc,
+				     char *argv[])
+{
+	int err;
+	int index = 0;
+	struct bt_le_per_adv_sync *per_adv_sync = NULL;
+
+	if (argc > 1) {
+		index = strtol(argv[1], NULL, 10);
+	}
```

Comment:
Instead of initialising `index` upon declaration I'd just add an else branch here. Makes the code easier to follow since you don't have to look that far away to see what the default value is.